tests/util.cpp

```diff
@@ -49,6 +49,52 @@ TEST_CASE("String", "[.util]") {
         dxvk::str::tonvss(ss, std::string("Longer-Than-Short-String-Longer-Than-Short-String-Longer-Than-Short-String"));
         REQUIRE_THAT(ss, SizeIs(64));
     }
+
+    SECTION("std::string") {
```

Comment:
`SECTION(“fromnullable”)`. Feel free to also updates the section name in the two function above.

Reply:
> Feel free to also updates the section name in the two functions above.

To match exactly the function name they test (so, `fromnvus` and `tonvss`)?

Reply:
Yep.